R/redshift-utils.R

```diff
@@ -571,7 +594,7 @@ temp_table_name <- function(n = 1, prefix = "tt_") {
 #' @export
 get_column_names <- function(con, table) {
   # This function gets column names from Redshift even if the table has no rows or is a temp table
-  if ("ident_q" %in% class(table)) {
```

Comment:
This is probably fine, but as I don't know why this was "ident_q" and not "ident", I'll flag this. Should `is_schema()` accept all three, "ident", "ident_q", "dbplyr_schema"?

Reply:
Good spot, I think you're right.